client/src/django.ts

```diff
@@ -50,10 +50,21 @@ const djangoClient = {
     const { data } = await apiClient.get('/configuration/');
     return data;
   },
+  async globalSettings() {
+    const { data } = await apiClient.get('/global/settings');
+    return data;
+  },
   async import(projectId: string) {
```

Comment:
Might be best to split this up into `globalImport()` and `projectImport(projectId: string)` to make it more explicit for callers.

Reply:
changed in f932555